sparkbox/stylelint-config-sparkbox
# chore: dependency and security updates

This PR address security vulnerabilities within yargs-parser, bl, lodash, and standard-version. It also updates stylelint and snyk packages.

## Validation
1. Checkout `master`.
1. Run `rm -rf node_modules && npm i`.
1. Run `npm audit`
- [x] Confirm 4 vulnerabilities require semver-major dependency updates.
1. Checkout chore--dependency-updates.
1. Run `rm -rf node_modules && npm i`.
1. Run `npm run test`
1. Run `npm audit`
- [x] Confirm 0 vulnerabilities are listed.
